docs/data-sources/cluster.md

```diff
@@ -1,22 +1,16 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "pcluster_cluster Data Source - pcluster"
+page_title: "aws-parallelcluster_cluster Data Source - aws-parallelcluster"
```

Comment:
Is this supposed to be `aws-parallelcluster_cluster`?

Reply:
This is Terraform standard: `[PROVIDERNAME]_[RESOURCE]`